.github/workflows/ci.yml

```diff
@@ -9,37 +9,42 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - name: Fetch the src
-      uses: actions/checkout@v3
-    - name: Set up Python 3.11
-      uses: actions/setup-python@v4
-      with:
-        python-version: 3.11
+      - name: Fetch the src
```

Comment:
Let's not make formatting changes in the same place where functional changes are made. It generates a hard to review diff every time this happens. Never do so. Also, if you want to lint YAML, make use of https://github.com/cherrypy/cheroot/blob/main/.yamllint config. This of course, must be done separately from any functional changes.

Reply:
This formatting changes was made automatically. I didn't notice that.

Reply:
You need to understand your development environment configuration and be in control of it. Check your editor and separate the changes.
Also, check what you commit. Using `git commit -v` will show you the preview of what is going into the patch. Check the PR's files tab too — there's no need for an external reviewer to be checking this for you since you can spot it faster and save yourself some time by reducing the number of review rounds.